src/Process/SkipLintProcess.php

```diff
@@ -23,15 +23,12 @@ class SkipLintProcess extends PhpProcess
     public function __construct(PhpExecutable $phpExecutable, array $filesToCheck)
     {
         $scriptPath = __DIR__ . '/../../bin/skip-linting.php';
```

Comment:
Brain fart (untested): changing this line to the below and removing the call to `realpath()` _might_ fix the issue with the PHAR ?
```suggestion
        $scriptPath = __DIR__ . '/../../bin/skip-linting.php';
        $scriptPath = dirname(dirname(__DIR__)) . '/bin/skip-linting.php';
```

Reply:
found a working solution at https://php-tips.readthedocs.io/en/latest/tips/run_any_phar_file.html